core/chains/evm/gas/l2_suggested_estimator.go

```diff
@@ -97,7 +98,10 @@ func (o *l2SuggestedEstimator) refreshPrice() (t *time.Timer) {
 	t = time.NewTimer(utils.WithJitter(o.pollPeriod))
 
 	var res hexutil.Big
-	if err := o.client.Call(&res, "eth_gasPrice"); err != nil {
+	ctx, cancel := evmclient.ContextWithDefaultTimeoutFromChan(o.chStop)
```

Comment:
Not sure you need this "new" function to be exposed, you can always use it directly from `utils` package.

Reply:
The reason for exposing it was because the default query timeout is only defined inside of the evmclient module.  Things outside that module shouldn't redefine the default themselves because then any updates to the default would have to be made in multiple places in the code.

Maybe we should just get rid of both `WithDefaultTimeout()` and `ContextWithDefaultTimeoutFromChan()`, instead exposing `queryTimeout` directly?  Then callers could create their own contexts however they want, just passing that in as a param.

Reply:
Oh I see now, as long as other reviewers agree, the current version is Okay with me.